File: Modul 3/Soal 4/PRAK304-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    int angka;

    printf("Masukkan bilangan cacah (0-99): ");
    scanf("%d", &angka);

    if (angka < 0 || angka >= 100) {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    } else if (angka == 0) {
        printf("Nol\n");
    } else if (angka < 10) {
        printf("Satuan\n");
    } else if (angka < 20) {
        printf("Belasan\n");
    } else {
        printf("Puluhan\n");
    }

    return 0;
}

File: Modul-2/Soal 4/PRAK304-2410817210014-ZAVIER GHANI.py
def ejaan_bilangan():

    angka = int(input("Masukkan bilangan cacah (0-99): "))

    if angka < 0 or angka >= 100:
        print("Anda Menginput Melebihi Limit Bilangan")
    elif angka == 0:
        print("Nol")
    elif angka < 10:
        print("Satuan")
    elif angka < 20:
        print("Belasan")
    else:
        print("Puluhan")

ejaan_bilangan()
